Tasks/Gradle/gradletask.ts

```diff
@@ -64,7 +66,7 @@ if (isCodeCoverageOpted && inputTasks.indexOf('clean') == -1) {
     gb.arg('clean'); //if user opts for code coverage, we append clean functionality to make sure any uninstrumented class files are removed
 }
 
-gb.argString(tl.getInput('options', false));
```

Comment:
Looking at the implementation of them "arg" takes array of strings or "single string with space separation" as well. Apparently, the options are space separated strings which doesn't require breaks. Correct me? :)